### ISSUE TITLE
Threat of Thassarian affecting offhand avoidance

### ISSUE THREAD

**Issue by Manpower99** [gamefreedomgit/Frostmourne#709]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**
As with classic confirming it, the offhand component of Frost DK's Abilities while Threat of Thassarian is talented should not be avoided whatsoever. this is exactly why fallen crusader enchant is used on the offhand primarily because the PPM is more reliable on offhand due to it never being avoided especially combined with lack of expertise.
Classic's Blight club discord confirms it in their theorycrafting along with guides written

warcraftlogs also for example :
https://classic.warcraftlogs.com/reports/gcbvkFHNKXnxdwh7#type=damage-done&source=16&fight=5&ability=-55268
1 dodge mainhand and offhand does not get dodged

https://classic.warcraftlogs.com/reports/Lr8KWqZvJ3gPF27t#type=damage-done&source=31&fight=3&ability=-51425
2 oblit dodges mainhand and none offhand

https://classic.warcraftlogs.com/reports/Lr8KWqZvJ3gPF27t#type=damage-done&fight=50&source=31&ability=-55268
one mainhand dodge none offhand

https://classic.warcraftlogs.com/reports/Lr8KWqZvJ3gPF27t#type=damage-done&fight=50&source=31&ability=-51425
same as above.

**How to reproduce:**

**How it should work:**
tldr: threat of thassarian should make Frost DK's affected abilities offhand component have 0 chance to be avoided

**Database links:**

**Comment by Manpower99** [comment]
After doing more testing with Revisited Skada by Bkader it seems this one is already implemented. other meter addons combine them so it looks like u had an offhand dodge.
